src/reporting/DisputeBondToken.sol

```diff
@@ -19,14 +19,14 @@ contract DisputeBondToken is DelegationTarget, Typed, Initializable, ERC20Basic,
     address private bondHolder;
     bytes32 private disputedPayoutDistributionHash;
     uint256 private bondRemainingToBePaidOut;
+    uint256 internal supply;
 
     function initialize(IMarket _market, address _bondHolder, uint256 _bondAmount, bytes32 _payoutDistributionHash) public beforeInitialized returns (bool) {
         endInitialization();
         market = _market;
         bondHolder = _bondHolder;
         disputedPayoutDistributionHash = _payoutDistributionHash;
         bondRemainingToBePaidOut = _bondAmount * 2;
-        totalSupply = 1;
```

Comment:
Since this inherits from ERC20Basic, I removed the totalSupply.  I take it this is ok to do since we are not attempting to make this an actual ERC20 token?

Reply:
Yeah, we'll be removing the ERC interface off this "soon".